g2p/tests/public/data/mic.psv

```diff
@@ -1,4 +1,13 @@
 mic|mic-ipa|tiꞌam|tiːɑm
 mic|mic-ipa|Miꞌkmaq|miːɡmɑx
+mic|mic-ipa|mi'kmaq|miːɡmɑx
 mic|mic-ipa|sqolj|əsxoltʃ
-mic|mic-ipa|sq|səx
\ No newline at end of file
+mic|mic-ipa|sq|səx
+
+# Test that ' is tokenized correctly as part of the words.
```

Comment:
Comments and blank lines are now supported in `public/data/*.?sv` test data files. I think it'll be nice to document why some weird test cases are being added.